signing/vrfsigner.go

```diff
@@ -17,7 +17,7 @@ type VRFSigner struct {
 
 // Sign signs a message for VRF purposes
 func (s VRFSigner) Sign(msg []byte) []byte {
-	return ed25519.Sign(s.privateKey, msg)
+	return ed25519.Sign2(s.privateKey, msg)
```

Comment:
this is actually the reason why i am seeing issues in the devnet. i didn't notice that we can't extract key from vrf signatures. i used ed signer in multi actor test and this is why it passed

tbh i think we should remove vrf signer, this is the same as ed signer but initialized with different key

Reply:
maybe vrf signer wasn't using ed++ by intention?

Reply:
Let's clarify that with research team

Reply:
noam has responded that we intentionally didn't want to use ed++